eclib/JModel_x86.ec

```diff
@@ -1,7 +1,8 @@
 (* -------------------------------------------------------------------- *)
 require import AllCore BitEncoding IntDiv SmtMap Ring List StdOrder Bool.
-(*---*) import CoreMap Map Ring.IntID IntOrder.
-require export JModel_common JArray JWord_array JMemory AES.
+(*---*) import CoreMap Map Ring.IntID IntOrder .
+require export JModel_common JArray JWord_array Jslh JMemory AES.
+export SLH64.
```

Comment:
What happens when a theory `require import` both `JModel_x86` and `JModel_m4`?

Reply:
Santiago, will fix this: we remove export SLH64 and SLH32 from JModel_x86 and JModel_m4.
The extraction will add a import SLH64 or SLH32 in the header depending on the architecture

Reply:
Done.